src/friendly-iframe-embed.js

```diff
@@ -132,13 +132,15 @@ export function setFriendlyIframeEmbedVisible(embed, visible) {
  * @param {!Element} container
  * @param {!FriendlyIframeSpec} spec
  * @param {function(!Window, ?./service/ampdoc-impl.AmpDoc=)=} opt_preinstallCallback
+ * @param {boolean=} isInUnsignedExp
  * @return {!Promise<!FriendlyIframeEmbed>}
  */
 export function installFriendlyIframeEmbed(
   iframe,
   container,
   spec,
-  opt_preinstallCallback // TODO(#22733): remove "window" argument.
+  opt_preinstallCallback, // TODO(#22733): remove "window" argument.
+  isInUnsignedExp
```

Comment:
how about control the exp outside this function by passing in a `spec` with `spec.html=null`

Reply:
Done.